tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py

```diff
@@ -27,55 +25,30 @@
 @tier4a
 class TestAddCapacityNodeRestart(ManageTest):
     """
-    Test add capacity when one of the nodes got restart
-    in the middle of the process. Don't forget to include the flag '--cluster-name' when running the test
+    Test add capacity when one of the worker nodes got restart
+    in the middle of the process.
     """
-    num_of_pvcs = 3
-    pvc_size = 100
 
     def test_add_capacity_node_restart(
-        self, nodes, multi_pvc_factory, pod_factory, workload_storageutilization_rbd,
-        node_type, num_of_nodes,
+        self, nodes, multi_pvc_factory, pod_factory,
+        workload_storageutilization_rbd, num_of_nodes,
     ):
         """
-        test add capacity when one of the nodes got restart in the middle of the process
+        test add capacity when one of the worker nodes got restart in the middle of the process
         """
         logging.info("Condition 1 to start the test is met: storageutilization is completed")
         # Please notice: When the branch 'wip-add-capacity-e_e' will be merged into master
         # the test will include more much data both before, and after calling 'add_capacity'function.
 
-        node_list = get_typed_nodes(node_type=node_type, num_of_nodes=num_of_nodes)
+        node_list = get_ocs_nodes(num_of_nodes=num_of_nodes)
         assert node_list, "Condition 2 to start test failed: No node to restart"
 
-        max_osds = 12
+        max_osds = 15
```

Comment:
Shouldn't this be calculated dynamically based on num of OCS nodes?
Also, this calculation needs to consider the deployments with less number of CPU such as when using Amazon EC2 i3en or the setup deployed with less resources.
@hnallurv @ebenahar how do we plan to handle such cases when a cluster is deployed with less resources.

Reply:
@sidhant-agrawal good point. 
@yitzhak12 Could you please report an OCS-CI issue about this matter. We will track it there and proceed with this PR as is.

Reply:
yes, I will create a new ocs-ci issue about this.